test/tstInterpDetailsMLSCoefficients.cpp

```diff
@@ -43,6 +45,12 @@ interpolate(ExecutionSpace const &space, SourceValues const &source_values,
 
 BOOST_AUTO_TEST_CASE_TEMPLATE(mls_coefficients, DeviceType, ARBORX_DEVICE_TYPES)
 {
+#ifdef KOKKOS_ENABLE_HIP
+  if (std::is_same_v<typename DeviceType::execution_space, Kokkos::HIP>)
```

Comment:
Please add a `FIXME_HIP` comment to all tests's disabled HIP places (can use `[ci skip]`) that would provide the context of why the tests are ignored for HIP, and what is required to reenable them.